.github/workflows/nixpkgs-review.yml

```diff
@@ -191,6 +197,66 @@ jobs:
           PR: ${{ github.event.inputs.pr }}
         run: cat ~/.cache/nixpkgs-review/pr-$PR/report.md >> $GITHUB_STEP_SUMMARY
 
+  build-aarch64-linux:
+    name: 'aarch64-linux'
+    if: ${{ github.event.inputs.aarch64-linux == 'true' }}
+    runs-on: ubuntu-24.04-arm
+
+    steps:
+      # - name: Free up space
+      #   uses: lucasew/action-i-only-care-about-nix@main
+      #   if: ${{ github.event.inputs.free-space == 'true' }}
```

Comment:
not working with aarch64-linux runners, opened lucasew/action-i-only-care-about-nix#1